drltrace/CMakeLists.txt

```diff
@@ -117,10 +117,33 @@ copy_and_adjust_drpaths(${CMAKE_RUNTIME_OUTPUT_DIRECTORY} drltrace)
 # XXX i#1960: add more tests for drltrace that check the output contains
 # expected library calls.
 #
-
 if (BUILD_TOOL_TESTS)
   get_target_path_for_execution(drltrace_path drltrace)
   get_target_path_for_execution(app_path drsyscall_app)
   prefix_cmd_if_necessary(drltrace_path OFF ${drltrace_path})
+
   add_test(drltrace ${drltrace_path} -- ${app_path})
+  add_test(drltrace_libcalls ${drltrace_path} -logdir - -print_ret_addr -- ${app_path})
+  add_test(drltrace_symargs ${drltrace_path} -logdir - -num_max_args 4 -- ${app_path})
+
+  #regex strings for libcalls and arguments printing test
+  set(libcall_args1_01 "    arg 0: 0x([0-9a-f]+)\n    arg 1: 0x([0-9a-f]+)\n")
+  set(libcall_ret "    and return to module id:([0-9]+), offset:0x([0-9a-f]+)\n")
+if (WIN32)
+  set(libcall_args2_0 "    arg 0: 0x([0-9a-f]+) \\(type=HANDLE, size=0x([0-9a-f]+)\\)\n")
+  set(libcall_args2_1 "    arg 1: 0x([0-9a-f]+) \\(type=int, size=0x([0-9a-f]+)\\)\n")
+
+  set(libcall_name1 "~~([0-9a-f]+)~~ KERNELBASE.dll!VirtualQuery\n")
+  set(libcall_name2 "~~([0-9a-f]+)~~ ntdll.dll!ZwQueryInformationProcess\n")
```

Comment:
So we test that drltrace is printing out one syscall's 1st 2 args and that they are of a certain type.  OK, seems a reasonable starting point.